package.json

```diff
@@ -54,7 +54,7 @@
     "test:ui:runner": "echo 'use `node scripts/functional_test_runner`' && false",
     "test:server": "grunt test:server",
     "test:coverage": "grunt test:coverage",
-    "checkLicenses": "grunt licenses",
+    "checkLicenses": "grunt licenses --dev",
```

Comment:
I believe we still want development licenses to be included by default in the license check per https://github.com/elastic/kibana/issues/19502

> While we do want to do license checking in Kibana for all dependencies, the release license report should only include dependencies that actually ship with Kibana.

Reply:
Agreed.  I was surprised with epixa's comment below saying this was already happening, but I was able to verify that [here](https://github.com/elastic/kibana/blob/master/src/dev/npm/installed_packages.js#L69) and [here](https://github.com/elastic/kibana/blob/master/tasks/licenses.js#L33).

I opened https://github.com/elastic/kibana/issues/19674.